sre-recipes/recipes/currencies_recipe/currencies_recipe.py

```diff
@@ -57,13 +77,28 @@ def _deploy_state(state):
         availability_command = "kubectl wait --for=condition=available --timeout=600s deployment/frontend"
         Recipe._run_command(availability_command)
 
+        if loadgen_ip and state:
+            # If we are breaking the service, start generating loads to the 
+            # frontend, after it is available, in order to expose the high 
+            # latency (caused by this recipe) in the frontend metrics. 
+            # The load will be generated by 20 users at spawn rate of 5 
+            # users/seconds, and the load will auto-stop after 10 minutes.
+            logging.info(f"Trying to start load generation")
+            resp = requests.post(
+                f"http://{loadgen_ip}:81/api/spawn/BasicHomePageViewingUser",
+                {
+                    "user_count": 20,
+                    "spawn_rate": 5,
+                    "stop_after": 600
+                })
+            logging.info(f"Load Generator API Response: {resp.text}")
+
```

Comment:
This is starting to be a really long function. Consider breaking it up at some point

Reply:
I will in next PR when I refactor SRE recipes to be YMAL config based